src/components/common/BalanceChannelsModal.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { PercentageOutlined, ReloadOutlined } from '@ant-design/icons';
+import { Button, Col, Modal, Row, Slider } from 'antd';
+import { usePrefixedTranslation } from 'hooks';
+import { useStoreActions, useStoreState } from 'store';
+import { ChannelInfo, Network } from 'types';
+import { format } from 'utils/units';
+
+interface Props {
+  network: Network;
+}
+
+const BalanceChannelsModal: React.FC<Props> = ({ network }) => {
+  const { l } = usePrefixedTranslation('cmps.common.BalanceChannelsModal');
+  const { visible, channelsInfo } = useStoreState(s => s.modals.balanceChannels);
+  const {
+    resetBalanceChannels,
+    hideBalanceChannels,
+    autoBalanceChannels,
+    manualBalanceChannels,
+    updateBalanceOfChannels,
+  } = useStoreActions(s => s.modals);
+
+  const CustomModalFooter = (
+    <Row gutter={10} justify="end">
+      <Col>
+        <Button onClick={() => hideBalanceChannels()}>{l('close')}</Button>
+      </Col>
+      <Col>
+        <Button onClick={() => updateBalanceOfChannels(network)}>{l('update')}</Button>
+      </Col>
+    </Row>
+  );
```

Comment:
Since you removed the custom font, you shouldn't need this custom footer any longer. This should make the Update button have the primary styling.